src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -174,20 +176,20 @@ private void createDatePanel() {
         TitledBorder tb = new TitledBorder(new LineBorder(Color.darkGray),                                        
                                            I18nUtils.getLocale()
          									+
-         									" for Today: ");
+         									" for Today:  ");
         datePanel_.setBorder(tb);
-        datePanel_.setToolTipText("Displays today's date and time.");
+        datePanel_.setToolTipText("Displays today's date & time. ");
 
-        JLabel dateLabel = new JLabel("Today's Date:");       
+        JLabel dateLabel = new JLabel("Today's Date: ");       
 
 //         String template = "Time: {0}";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3ACommentedOutCodeLine)